GencallFormat.py

```diff
@@ -55,6 +56,8 @@ def generate_sample_format_info(self, bpm_records, vcf_record, sample_name):
             sample_name (string): Name of sample
 
         Returns:
-            float: Minimum GenCall score (or None for empty arguments)
+            int: Minimum GenCall score, encoded as a phred quality (or None for empty arguments)
         """
-        return self._get_min_gencall_score(bpm_records)
+        min_gencall_score = self._get_min_gencall_score(bpm_records)
+        phred_quality_score = int(round(-10*log10(min(1.0, max(0.00001, 1 - min_gencall_score)))))
```

Comment:
nvm I see the conversion here but not sure why an `int` conversion is required?

Reply:
you're right I don't think it needs it, will fix

Reply:
Nvm apparently it does. Prints one decimal place even though the type should be an int, not entirely sure without digging deeper but it passes the regression tests the way it is